en/docs/learn/api-security/threat-protection/bot-detection.md

```diff
@@ -213,8 +212,8 @@ Follow the instructions below to view the bot detection data for the unauthorize
 
      `https://<IP_Address>:9443/admin`
 
-8. Click **BOT DETECTION DATA**.
+8. Click **Bot Detection Data**.
 
      ![Bot detection data details for unauthorized API calls]({{base_path}}/assets/img/learn/bot-data.png)
 
-    If a hacker (e.g., bot attacker) tries to invoke an open service API, the Bot detection data details, which appear in the `<API-M_HOME>/repository/logs/wso2-BotDetectedData.log` trace log, will appear.
+    If a hacker (e.g., bot attacker) tries to invoke an open service API, the Bot detection data details, which appear in the `<API-M_HOME>/repository/logs/wso2-BotDetectedData.log`.
```

Comment:
If a hacker (e.g., bot attacker) tries to invoke an open service API, the Bot detection data details will appear in the `<API-M_HOME>/repository/logs/wso2-BotDetectedData.log` file.

WDYT?

Reply:
Much clear.
fixed with: https://github.com/wso2/docs-apim/pull/1665/commits/24b9f2f22005a74c3994372ea2ba1cbef8c26f63